### DISCUSSION TITLE
Análise de um código (Google Pixel)

### DISCUSSION THREAD

**Discussion by ghost** [frontendbr/forum#575] [Discussão]
Depois de bater um papo sobre semântica com um amigo, ele me passou o site do Google Pixel e me perguntou quantas tag _h (h1, h2, h3...)_ havia no site. Além da ausência de outras tags que são comuns no desenvolvimento web.

Pois bem, queria entender a lógica de usar _div_ para tudo, ou quase tudo (salvo os links). E também no que isso impacta no SEO, se é que impacta em algo hoje em dia.

Creio que seria uma facilidade na hora de desenvolver, daríamos ainda mais importância para as classes. Vou deixar abaixo o link e pedir que deem uma olhada. 😃 

https://madeby.google.com/phone/

**Comment by rubenmarcus** [comment]
Não sei se o fato de o site ser feito pelo próprio Google, escaparia as regras de semântica que o proprio buscador usa para rankear os sites...

Mas tem sites feitos no WIX , que abusam bastante de divs, e estão bem rankeados também.

**Comment by rubenmarcus** [comment]
alias esse site e feito no Wix e esta semanticamente melhor do que o que você postou : https://www.mancity.com/

**Comment by willycamargo** [comment]
Acredito que se o google (buscador) for levar em consideração apenas a semântica do código para formar seu ranking, certamente a primeira página seria uma loucura.

Não tenho dados técnicos para afirmar isso e nem pesquisei o suficiente, mas:
IMO, o google (google.com) é muito mais que apenas um buscador, ele esta cada dia mais inteligente e é diretamente integrado com o Google Assistant, o que permite personalizar os resultados de buscar de acordo com a vontade/escolha do usuário que esta buscando.

Fiz alguns poucos testes práticos pesquisando alguns termos no meu notebook (com meu usuário) e os mesmos termos no computador de minha mãe (com o usuário dela). Os resultados tiveram suaves diferenças, mas suficientes para influenciar na escolha de quem esta pesquisando.

Portanto acredito que o conteúdo e a relevância dele para o usuário é mais importante do que a semântica em si.
